jenkins.check_copyright.sh

```diff
@@ -32,7 +32,7 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
     grep -v -E '\.def$' |
     grep -v -E '\.inc$' |
     grep -v -E 'test/benchmarks/.*\.js$' |
-    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; if [ \$? -ne 0 ]; then cat $ERRFILETEMP >> $ERRFILE; fi"
+    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; || cat $ERRFILETEMP >> $ERRFILE; fi"
```

Comment:
You can also do `git commit --amend` and then `git push --force`

@goyakin that's what I thought too, but the check ran this script and didn't complain. I wonder if it just quietly ignored that line.

At any rate I'm fairly certain the logic would be wrong because the `||` doesn't apply to the result of the previous statement, so it should be fixed in any case.